lib/mix/lib/mix/tasks/test.ex

```diff
@@ -156,6 +156,8 @@ defmodule Mix.Tasks.Test do
       external commands which produce output on stdout upon file system modification
     * `--max-cases` - sets the maximum number of tests running async. Only tests from
       different modules run in parallel. Defaults to twice the number of cores
+    * `--max-failures` - maximum number of tests that may fail. When the maximum is reached, the
+       execution of tests is stopped and the results are returned.
```

Comment:
Minor nitpick: looking at other items, I think this line should not end with a dot `.` for consistency.